app/views/caseload.html

```diff
@@ -0,0 +1,58 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+{% include "includes/breadcrumbs.html" %}
+
+{% include "includes/validation-error-messages.html" %}
+
+{% include "includes/title-subtitle.html" %}
+
+{% include "includes/sub_nav.html" %}
+
+<h3 class="heading-medium">Overall caseload by tier</h3>
+
+<table class="sln-table-caseload data-table">
+  <thead>
+    <tr>
+      <th colspan="2">Offender manager</th>
+      <th colspan="9">Case tiers</th>
```

Comment:
Pa11y is gonna cry about this, need to have an id and a header inside all your <th> tags.  You can look at the organisation-overview.html file to see how its done